python-backend/app/__init__.py

```diff
@@ -25,10 +26,14 @@ def create_app(test_config=None):
     else:
         # load the test config if passed in
         app.config.from_object(test_config)
+    
+    configure_uploads(app, documents)
```

Comment:
Maybe it's a mixed tabs/spaces issue? Things in this block aren't lined up with the existing code